.gitignore

```diff
@@ -12,3 +12,4 @@ gtdb_attribs.jsonl
 .dockerignore
 *_pycache__
 gtdb_attribs2.jsonl
+.DS_Store
```

Comment:
I don't really understand the reason for moving the files below from `genome_attribs` to `genome_collection`. The former seems fine to me - what's the rationale?

Reply:
I was thinking taxa_count is not exactly genome_attributes.

Reply:
Ok, I guess we could split that into 2 directories, but this is fine for now, at least.
:+1: